contracts/Auction.sol

```diff
@@ -148,10 +141,15 @@ contract Auction is Permissioned {
     afterAuctionEnds
     auctionNotEnded
     {
-        winnerAddress = ConfAddress.unsafeToAddress(highestBidder);
+        winnerAddress = FHE.decrypt(highestBidder);
         if (winnerAddress == address(0)) {
-          winnerAddress = NO_BID_ADDRESS;
+            winnerAddress = NO_BID_ADDRESS;
+        } else {
+            _wfhenix.transferEncrypted(auctioneer, highestBid);
```

Comment:
added here the transfer of the winning bid to the auctioneer, which did not exist before